src/base_station/gui.cpp

```diff
@@ -832,6 +965,81 @@ void do_help_menu(std::vector<const char*> commands, std::vector<const char*> de
     help_layout.pop();
 }
 
+void do_image_display(char *firstImageName, char *secondImageName, Session *bs_session){
```

Comment:
I have changed the function to only allocate 1 texture ID to the images rather than repeatedly assign them new ID's. This seems to have fixed the memory issue.